src/18_lines.py

```diff
@@ -0,0 +1,28 @@
+BOARD_SIZE = 8
+
+
+def under_attack(col, queens):
```

Comment:
here you have 2 empty lines between constant and code, earlier you put 0 lines. Styling should be consistent.